bin/resources/GameIndex.yaml

```diff
@@ -5553,7 +5549,7 @@ SCPS-15078:
   name: "EyeToy - Uki Uki Panic [Game Only]"
   region: "NTSC-J"
 SCPS-15079:
-  name: "Bakufuu Slash! Kizna Arashi [with Camera]"
+  name: "Bakufuu Slash! Kizuna Arashi [EyeToy Camera Doukonban]"
```

Comment:
I was SURE it was a typo because of the infamous Japenglish (and it is somehow but I didin'y know they've added KIzna in romajis). Sorry about that one. u_u
Reverted.

Reply:
Thanks :)